### PR TITLE
[wheel] cut "cpp" out of "all"

### PR DESCRIPTION
**Pull request by aslonnie**
all will not include "cpp" anymore; use "allcpp" instead if "cpp" is required. "all" will just be "all python".

### CODE REVIEW

**Comment by aslonnie** [comment]
context: "ray-cpp" on pypi is running out of space; so we realized that it is better to not have a hard dependency on "ray-cpp".

**Review by jjyao — APPROVED** [review]

File: ci/docker/min.build.Dockerfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ MINIMAL_INSTALL=1 PYTHON=${PYTHON_VERSION} ci/env/install-dependencies.sh
 rm -rf python/ray/thirdparty_files
 
 # install test requirements
-python -m pip install -U pytest==7.0.1 pip-tools==7.3.0
+python -m pip install -U pytest==7.0.1 pytest-asyncio==0.16.0 pip-tools==7.3.0
```
Comment (can-anyscale) [inline ci/docker/min.build.Dockerfile:29]: this is to fix the aiohttp issue?

**Review by can-anyscale — APPROVED** [review]

File: ci/docker/min.build.Dockerfile (hunk 1 above)
Comment (aslonnie) [reply ci/docker/min.build.Dockerfile:29]: not really. this is that there are several tests in the test suite that is async, and they are skipped. I was trying to understand if it has anything to do with the event loop thing.

will revert this.

Comment (aslonnie) [reply ci/docker/min.build.Dockerfile:29]: we probably should add this.. but I can do it in another PR.

**Pull request merged**